file_store/src/heartbeat.rs

```diff
@@ -76,6 +79,51 @@ impl MsgTimestamp<Result<DateTime<Utc>>> for CellHeartbeatIngestReportV1 {
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct ValidatedHeartbeat {
+    pub cbsd_id: String,
+    pub pub_key: PublicKeyBinary,
+    pub reward_multiplier: f32,
```

Comment:
should we be keeping all the numbers that factor into rewards calculations be kept as integers or decimals? we've allowed a _few_ floats but I think those have so far always been relative to things like transmit values and other RF-things